app/models/service_provider_request.rb

```diff
@@ -12,15 +11,14 @@ def initialize(
     ial: nil,
     aal: nil,
     requested_attributes: [],
-    biometric_comparison_required: false
+    biometric_comparison_required: false # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
for my understanding, this is the one line that is not completely reverted?

Reply:
That is correct. I kept the revert changes in the first commit and then added this in a second.